charters/README.md

```diff
@@ -0,0 +1,18 @@
+# Charter Guide
+
+Each HCA DCP project must have a charter that specifies its scope, responsibilities, and leadership roles.
+
+The charter process provides a simple and consistent path for new projects to be formed that align with the direction of the HCA DCP roadmap and its priorities.
+
+
+## What the process is
+  - Fork the HumanCellAtlas dcp-community repository
+  - Make a subdirectory named charters/`MYPROJECTNAME`
+  - Copy `charters/charter-template.md` to `charters/MYPROJECTNAME/charter.md`
+  - Fill in the charter with attention to detail
+  - Submit a pull request. Add the "charter-proposed" label
+  - Share a link to the pull request for community review on the HumanCellAtlas **#dcp** slack channel
+  - Feedback is added to the pull request comment thread by community members
+  - Update the pull request in response to feedback
+  - Share a link to the pull request for approval on the HumanCellAtlas **#tech-architecture** slack channel when all feedback is addressed. Replace "charter-proposed" with "charter-final-review". 
+  - The request will be discussed at the next meeting of the Technical Leads. If approved, the Leads replace "charter-final-review" with "charter-approved" and merge the pull request. If rejected, the Leads replace "charter-final-review" with "charter-rejected" and add a rationale to the pull request comment thread.
```

Comment:
@brianraymor This is the "Bi-weekly DCP PM meeting" every other Thursday at 8am pacific, right?  Once this PR is merged, it would be good to update the agenda of this meeting to point out the PMs/Tech Leads need to approve charters in that meeting.  What about RFCs?  I can hold that question, though, until you have a PR to tackle that SOP.  I'm assuming the RFC process documentation will go in this repo, is that right?

Reply:
No. I'm referring to the bi-weekly _Tech Architecture_ meeting.  Technical charters are approved by the Tech Leads. If we have process charters, those charter would be approved by PM. 

The RFC process will also be in the community repo. One stop shopping.